database/migrations/2018_08_15_103716_move_photos.php

```diff
@@ -24,59 +24,62 @@ public function up()
 				$results = DB::table(env('DB_OLD_LYCHEE_PREFIX', '') . 'lychee_photos')->select('*')->orderBy('id', 'asc')->orderBy('album', 'asc')->get();
```

Comment:
This will need testing proper testing.

Reply:
I actually have a old v3 db locally to test that :)

Reply:
Feel free to do it. :joy: